frontend/src/metabase/styled-components/theme/css-variables.ts

```diff
@@ -22,22 +22,21 @@ type MetabaseComponentThemeKey = FlattenObjectKeys<MetabaseComponentTheme>;
  */
 export function getMetabaseCssVariables(theme: MantineTheme) {
   return css`
-    ${getDesignSystemCssVariables(theme)}
-    ${getThemeSpecificCssVariables(theme)}
+    :root {
```

Comment:
Put our CSS custom properties in `:root`, so it works with other colors defined in `:root` from `colors.module.css`.

Then we don't have to add new colors here when we add new colors to `colors.module.css`